bucoffea/plot/stack_plot.py

```diff
@@ -246,19 +256,19 @@ def make_plot(acc, region, distribution, year,  data, mc, signal=None, outdir='.
         ax.set_xlim(xlim[0],xlim[1])
     elif 'xlim' in plot_settings.keys():
         ax.set_xlim(plot_settings['xlim'])
-
-    if rylim:
-        rax.set_ylim(*rylim)
-    else:
-        rax.set_ylim(0.75,1.25)
-    loc1 = matplotlib.ticker.MultipleLocator(base=0.2)
-    loc2 = matplotlib.ticker.MultipleLocator(base=0.1)
-    rax.yaxis.set_major_locator(loc1)
-    rax.yaxis.set_minor_locator(loc2)
-    rax.grid(axis='y',which='minor',linestyle='--')
-    rax.grid(axis='y',which='major',linestyle='--')
-    ax.set_ylabel('Events / Bin width')
-    rax.set_ylabel('Data / MC')
+    
```

Comment:
Let's just have a 'ratio=True / False' argument that controls whether a ratio plot is drawn.